ApollosProject/apollos-apps
# Expose topics settings

## Exposed topics in user settings per https://3.basecamp.com/3926363/buckets/27088350/todos/5737403621.

## 🔬 To Test
As of opening the PR, tests are not yet written. 

Ideas for testing
- Verify the Topics page launches when button pressed
- Verify button is shown when (and only when) tags are available.

## 📸 Screenshots

Figma design: 

![](https://storage.3.basecamp.com/3926363/blobs/0075fec2-9698-11ed-9d0a-b2d1a80aa527/download/CleanShot%202023-01-17%20at%2013.51.42.png)

Code implementation:
![Screen Shot 2023-01-30 at 10 57 10 AM](https://user-images.githubusercontent.com/45675751/215543033-6d3eea86-d532-4627-a2cf-d7ae579f9081.png)

## Checklist
